### PR TITLE
Added tests for defaults in URL and File engine

### LINKED ISSUE #5666: DEFAULT expressions not evaluated for File engine
```sql
echo '{"a":1}' > db/data/default/json/data.JSONEachRow

create table json(a int, b int default 7) engine File(JSONEachRow);
set input_format_defaults_for_omitted_fields = 1;
select * from json;
┌─a─┬─b─┐
│ 1 │ 0 │
└───┴───┘ 
```

When fixing, make sure it also works for HDFS and URL.

### PR DESCRIPTION
**Pull request by nikitamikhaylov**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Build/Testing/Packaging Improvement

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Added tests for defaults in URL and File engine. This closes #5666

### CODE REVIEW

File: tests/queries/0_stateless/01622_defaults_for_file_engine.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+insert into table function file("data1622.json", "TSV", "value String") VALUES ('{"a":1}');
+drop table if exists json;
+create table json(a int, b int default 7) engine File(JSONEachRow, 'data1622.json');
```
Comment (alexey-milovidov) [inline tests/queries/0_stateless/01622_defaults_for_file_engine.sql:3]: Maybe also add `c default a + b`? Just to be sure :)

File: tests/queries/0_stateless/01622_defaults_for_url_engine.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+
+PORT="$(($RANDOM%63000+2001))"
+
+TEMP_FILE="$CURDIR/01622_defaults_for_url_engine.tmp"
+
+function thread1
+{
+    while true; do 
+        echo -e "HTTP/1.1 200 OK\n\n{\"a\": 1}" | nc -l -p $1 -q 1;
```
Comment (alexey-milovidov) [inline tests/queries/0_stateless/01622_defaults_for_url_engine.sh:15]: Nice.

BTW, also it's possible to query ClickHouse itself by url.

**Review by alexey-milovidov — APPROVED** [review]

File: tests/queries/0_stateless/01622_defaults_for_file_engine.sql (hunk 1 above)
Comment (nikitamikhaylov) [reply tests/queries/0_stateless/01622_defaults_for_file_engine.sql]: Added

**Comment by alexey-milovidov** [comment]
> https://clickhouse-test-reports.s3.yandex.net/19015/1d9a3a492a21ca83662e473466c46e5aa1051669/integration_tests_(release).html#fail1

@CurtizJ

**Pull request merged**